canvas_modules/common-canvas/src/common-canvas/svg-canvas-d3.scss

```diff
@@ -747,15 +747,15 @@ g.bkg-col-cyan-50 {
 		font-weight: 500;
 	}
 	h4 {
-		font-size: 16px;
+		font-size: $spacing-05;
```

Comment:
You can revert all space tokens for fonts. Fonts need a specific size and weight for each. Space tokens are used for spacing and aligning objects, there are typography tokens for fonts but we wont need to use those in this case either as these are specifically set